src/main/resources/application.yaml

```diff
@@ -40,16 +40,16 @@ spring:
             issuer-uri: ${darts.azure.active-directory-b2c-base-uri}/${AAD_B2C_TENANT_ID_KEY:00000000-0000-0000-0000-000000000000}/v2.0/
             provider: external-azure-ad-provider
           internal-azure-ad:
-            client-id: ${AAD_B2C_CLIENT_ID_KEY:00000000-0000-0000-0000-000000000000}
-            client-secret: ${AAD_B2C_CLIENT_SECRET_KEY:}
+            client-id: ${AAD_CLIENT_ID:00000000-0000-0000-0000-000000000000}
+            client-secret: ${AAD_CLIENT_SECRET:}
             scope: openid
             redirect-uri: ${darts.portal.url}/auth/callback
             logout-redirect-uri: ${darts.portal.url}/auth/logout-callback
             authorization-grant-type: authorization_code
             response-type: code
             response-mode: form_post
             prompt: login
-            issuer-uri: ${darts.azure.active-directory-b2c-base-uri}/${AAD_B2C_TENANT_ID_KEY:00000000-0000-0000-0000-000000000000}/v2.0/
+            issuer-uri: https://login.microsoftonline.com/${AAD_TENANT_ID:00000000-0000-0000-0000-000000000000}/v2.0/
```

Comment:
the base URL should not be hardcoded like we discussed. It should come from property similar to B2C.